src/docs/file-formats.dox

```diff
@@ -6,7 +6,9 @@ WiredTiger supports three underlying file formats: row-store and
 column-store, with an underlying special case of column-store for
 bit-field values. All three formats are B+tree implementations of
 key/value stores. WiredTiger also supports @ref lsm, implemented as a
-tree of B+trees.
+tree of B+trees. In addition, there is experimental support for
+@ref tiered_storage, allowing a B+tree to span multiple files and
```

Comment:
And here's a 3rd form. B-tree, Btree and B+tree!

Reply:
We use "btree" and "B-Tree" so ubiquitously that I would eliminate "B+Tree", even though there is a reasonable case that our trees are more "B+ like".

Reply:
We're going with `btree`, that's been the dominant form.

Reply:
I'm leaving this one for WT-12956.  Sometimes we want to specifically state that the implementation is B+Tree, I'm not sure of this is one of those cases.